bdai_ros2_wrappers/bdai_ros2_wrappers/scope.py

```diff
@@ -476,13 +477,16 @@ def top(
     domain_id: typing.Optional[int] = None,
     **kwargs: typing.Any,
 ) -> typing.Iterator[ROSAwareScope]:
-    """
-    Manages a ROS 2 aware scope, handling ROS 2 context lifecycle as well.
+    """Manages a ROS 2 aware scope, handling ROS 2 context lifecycle as well.
 
     Args:
         args: optional command-line arguments for context initialization.
         context: optional context to manage. If none is provided, one will
         be created. For global scopes, the default context will be used.
+        context: An optional ros context.
```

Comment:
@amessing-bdai nit: context is documented twice

Reply:
Fixed.